src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">  
+        <img class="logo_img"
```

Comment:
<img width="1318" alt="Screenshot 2023-02-07 at 18 16 11" src="https://user-images.githubusercontent.com/53228259/217300989-34c860e1-6027-48a2-8831-33fe6b4d4e31.png">

Reply:
check all tags

Reply:
is not fixed